pkg/asset/manifests/template.go

```diff
@@ -1,6 +1,9 @@
 package manifests
 
-import "github.com/openshift/installer/pkg/types/baremetal"
+import (
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/installer/pkg/types/baremetal"
```

Comment:
Separate imports of installer packages from other packages.
```suggestion
	configv1 "github.com/openshift/api/config/v1"

	"github.com/openshift/installer/pkg/types/baremetal"
```